.github/workflows/build-gem.yml

```diff
@@ -0,0 +1,21 @@
+name: Check if Gem exists in production
```

Comment:
What that actually means is "Check if the current gem version is already released on rubygems.org"?

Reply:
Ah, just to make visible to someone opening a PR that the gem version already exists on rubygems.org.

It will check if the gem in the specific version exists already on rubygems. If it does, it should error, which would means that as soon this PR gets merged, we are going to try to repush the same version

Reply:
Nice, could we change the description as proposed? I think the term 'production' is pretty unclear here.

Reply:
done!